internal/bminventory/inventory.go

```diff
@@ -3204,11 +3204,20 @@ func (b *bareMetalInventory) DownloadClusterKubeconfig(ctx context.Context, para
 }
 
 func (b *bareMetalInventory) DownloadClusterKubeconfigInternal(ctx context.Context, params installer.DownloadClusterKubeconfigParams) (io.ReadCloser, int64, error) {
-	if err := b.checkFileForDownload(ctx, params.ClusterID.String(), constants.Kubeconfig); err != nil {
+	respBody, contentLength, err := b.downloadClusterFileIfAvailable(ctx, params.ClusterID.String(), constants.Kubeconfig)
```

Comment:
maybe better to verify if not exists? In case kubeconfig not exists does we log it? Can you verify that there is no error log in this case?